src/WebJobs.Script/Workers/Rpc/JobHostRpcWorkerChannelManager.cs

```diff
@@ -12,45 +13,86 @@ namespace Microsoft.Azure.WebJobs.Script.Workers.Rpc
     internal class JobHostRpcWorkerChannelManager : IJobHostRpcWorkerChannelManager
     {
         private readonly ILogger _logger;
-        private ConcurrentDictionary<string, IRpcWorkerChannel> _channels = new ConcurrentDictionary<string, IRpcWorkerChannel>();
+        private ConcurrentDictionary<string, ConcurrentDictionary<string, IRpcWorkerChannel>> _channels = new ConcurrentDictionary<string, ConcurrentDictionary<string, IRpcWorkerChannel>>();
```

Comment:
That's a complex nested generic class. Should it be a declared explicit class instead?